care/emr/migrations/0032_alter_observationdefinition_body_site_and_more.py

```
@@ -0,0 +1,72 @@
+        migrations.AlterField(
+            model_name='observationdefinition',
+            name='method',
+            field=models.JSONField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='observationdefinition',
+            name='permitted_unit',
+            field=models.JSONField(blank=True, null=True),
+        ),
+        migrations.CreateModel(
+            name='ActivityDefinition',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)